capture_metrics.go

```diff
@@ -81,8 +81,10 @@ func (m *Metrics) CaptureMetrics(w http.ResponseWriter, fn func(http.ResponseWri
 		}
 	)
 
+	defer func() {
+		m.Duration += time.Since(start)
+	}()
```

Comment:
You should add an inline comment about the fact this defer is there to increase the duration even if the fn panics